models/customers.sql

```diff
@@ -38,7 +38,8 @@ customer_payments as (
 
     from payments
 
-    left join orders using (order_id)
+    left join orders on
+         payments.order_id = orders.order_id
```

Comment:
```suggestion
    left join orders
         on payments.order_id = orders.order_id
```

Let's make the indentations consistent